modules/p2/ding_user_consent/ding_user_consent.install

```diff
@@ -5,6 +5,24 @@
  * Setup the fields needed for ding list.
  */
 
+/**
+ * Implements hook_uninstall().
+ */
+function ding_user_consent_uninstall() {
+  $url = variable_get('ding_user_consent_loan_history_store_description_link', '');
+  if ($url) {
+    list(, $nid) = explode('/', $url);
```

Comment:
Should be be explicit that we expect/limit `explode` to return 2 elements?

Reply:
Well, it's explicit in that we only have two elements in `list()`, it'll just discard the rest. Safeguarding against the url having an extreme number of slashes in a variable set by the module itself seems a bit overkill to me.